experimental/GModule/src/GModule.jl

```diff
@@ -949,11 +1012,17 @@ end
 function _regular_gmodule(G::Group, M)
   ge = collect(G)
   ZG = gmodule(G, [hom(M, M, [M[findfirst(isequal(ge[i]*g), ge)] for i=1:length(ge)]) for g = gens(G)])
+  set_attribute!(ZG, :is_regular => ge)
```

Comment:
oh wait, so the `is_regular` attribute is *not* set to a bool, as it name would suggest, but rather to... whatever `collect(G)` produces? I.e. probably a list of all elements? That seems.. odd ?